src/styles/utils/_variables.scss

```
@@ -0,0 +1,16 @@
+$square-width: 100px;
+$square-height: 100px;
+$square-gap: 1px;
+$week-length: 7;
+$container-width:  #{($square-width + $square-gap) * $week-length};
+$min-month-length: 28;
+$max-month-length: 31;
+
+$week:
+  "mon" 0,
+  "tue" #{$square-width + $square-gap},
+  "wed" #{($square-width + $square-gap) * 2},
+  "thu" #{($square-width + $square-gap) * 3},
+  "fri" #{($square-width + $square-gap) * 4},
+  "sat" #{($square-width + $square-gap) * 5},
+  "sun" #{($square-width + $square-gap) * 6};
```

```suggestion
  "tue" $square-width + $square-gap,
  "wed" ($square-width + $square-gap) * 2,
  "thu" ($square-width + $square-gap) * 3,
  "fri" ($square-width + $square-gap) * 4,
  "sat" ($square-width + $square-gap) * 5,
  "sun" ($square-width + $square-gap) * 6;
```
You can to not use **#{}** here